resources/windows/msi/project.wxs.erb

```diff
@@ -0,0 +1,775 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi'
+  xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <!--
+  The Product Id should not be changed, the unique identifier enables
+  un-install of the MSI package easily.  More information about supporting
+  major upgrades is available at:
+  http://wix.sourceforge.net/manual-wix3/major_upgrade.htm
+  Name is made of localized product name and version number.
+  -->
+  <Product Id="$(var.ProductCode)" UpgradeCode="$(var.UpgradeCode)"
+    Name="$(var.OurProductName) Agent$(var.OurProductPlatformText)"
+    Language='1033'
+    Codepage='1252'
+    Version="$(var.ProductVersionNumber)"
+    Manufacturer="$(var.OurCompanyName)">
+
+    <Package
+      InstallerVersion='300'
+      InstallScope="perMachine"
+      Description="$(var.OurProductName)$(var.OurProductPlatformText) Installer"
+      Comments="http://www.puppetlabs.com/"
```

Comment:
@shermdog Yeah - quite a few of the "Wix" variables can be replaced with ERB variables.